crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -219,6 +219,10 @@ def families
         })
       end
     end
+
+    respond_to do |format|
+      format.html
+    end
```

Comment:
Can we get a comment at the beginning of the method to say that this is a UI-only method, not an API method?